network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html

```diff
@@ -7,13 +7,15 @@
   {% include "./fragments/primary_heroguts.html" with root=root page=page %}
 {% endblock %}
 
+{% block content_wrapper_class %}{% endblock %}
+
 {% block campaign_page_header %}{% endblock %}
 {% block campaign_page_menu %}{% endblock %}
 
 Note: Wrapper HTML elements are copied from primary_page_menu.html as we want menu on BanneredCampaignPage to look exactly the same as PrimaryPage's.
 
 {% block minisite_title %}
-  <div id="multipage-nav-wrapper" class="hidden-sm-down full-width mb-4">
+  <div id="multipage-horizontal-nav-wrapper" class="hidden-sm-down full-width">
```

Comment:
renamed this id name from `multipage-nav-wrapper` to `multipage-horizontal-nav-wrapper`